deploy/helm/kubecf/values.yaml

```diff
@@ -243,6 +243,110 @@ features:
       service:
         # This setting is not currently configurable and must be HIDDEN
         nodePort: 31666
+
+  # To support multi-clusters, the diego-cell would be deployed separately in another different cluster if these two flags, multiple-cluster and diego-segment, are true 
+  multiple_cluster_mode:
+    multiple_cluster:
```

Comment:
This stutters, can you please remove `multiple_cluster` and rename `multiple_cluster_mode` to `multi_cluster`?
I think the settings should look like:

```
multi_cluster:
  control_plane:
    enabled: false
  cell_segment:
    enabled: true
  ...
```

Reply:
okay, got it.  I will change it.